.github/workflows/release.yml

```diff
@@ -0,0 +1,75 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs: 
+      current_version:
```

Comment:
Want to get the current version from somewhere (Cargo.toml, CHANGELOG.md, etc).

Reply:
I think need to know the current release number for the release work.

The version number in `Cargo.toml` may change to a snapshot version and may not be a good value to get and use automatically.

It should be possible to assume that the release worker knows the current version number, even if unexpected release work occurs, such as when re-releasing a fix for a previous version due to a security fix.

Therefore, I think it's better to let the release worker enter the current version it knows.